CalculatorExercises/1-CalculatorBasic/Main.java

```diff
@@ -0,0 +1,25 @@
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+        
+        Scanner scan = new Scanner(System.in);
+
+        Calculator calculator = new Calculator();
+        
+        while(true) {
+
+            System.out.println(calculator.toString(calculator.getResult()));
+
+            String input = new String(scan.nextLine()); 
+
+            StringHelper.removeWhitespace(input);
```

Comment:
you have to assign result of that method, now it is never used!
input = StringHelper.removeWhitespace(input);